index.html

```diff
@@ -1,177 +1,241 @@
+    <div class="portfolio-summary">
+      <h1 class="title main-head">About Myself</h1>
+      <p class="description">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#" class="blue-text" class="connect-link">LET’S CONNECT</a>
+      <ul class="contact-list">
+        <li>
+          <a href="#">
+            <img src="icons/twitter-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/Linkedin-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/m-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/github-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="icons/Vector-icon.png" class="contact-list-icon" />
+          </a>
+        </li>
+      </ul>
+      <button class="portfolio-btn">Get my resume</button>
+    </div>
+    <ul class="skills">
+      <li class="skill">
+        <div class="skill-title">
+          <h2 class="domain-name">Languages</h2>
+          <img src="icons/down-arrow.png" class="more-icon" />
+        </div>
+        <ul class="languages-list">
+          <li class="language">
+            <img src="images/javascript-icon.png" class="language-icon" />
+            <h3 class="language-name">Javascript</h3>
+          </li>
+          <li class="language">
+            <img src="images/html-icon.png" class="language-icon" />
+            <h3 class="language-name">HTML</h3>
+          </li>
+          <li class="language">
+            <img src="images/css-icon.png" class="language-icon" />
+            <h3 class="language-name">CSS</h3>
+          </li>
         </ul>
-    </section>
+      </li>
+      <li class="unactive-skill">
+        <h3 class="domain-name">Frameworks</h3>
+        <img src="icons/right-arrow.png" class="more-icon" />
+      </li>
+      <li class="unactive-skill">
+        <h3 class="domain-name">Skills</h3>
+        <img src="icons/right-arrow.png" class="more-icon" />
+      </li>
+    </ul>
+  </section>
+  <footer class="portfolio-footer">
+    <form class="footer-form">
+      <h2 class="form-header">Contact me</h2>
+      <p class="footer-description">If you have an application you are interested in developing, a feature that you need
+        built or a project that needs
+        coding. I’d love to help with it</p>
+      <input type="text" class="form-input">
+      <input type="email" class="form-input">
+      <textarea class="form-textarea" placeholder="Write your message here"></textarea>
+      <button class="form-button">Get touch</button>
```

Comment:
- [ ] Your form looks very like the Figma template, but currently it is not working. Kindly see the 'get started' section of the [Formspree service](https://formspree.io/html/) and implement the form with the correct action and method to make it work.
 - [ ] The inputs should be required to prevent the user to submit the form with an empty input field. Please fix this issue to meet the project requirements.
 - [ ] The name and message inputs should have a maximum of 30 and 500 characters respectively according to the project requirements. Kindly add this validation attribute to meet the project requirements.
 - [ ] According to the project requirements, all the fields should have placeholder text, and currently, only the message field has one. Please implement some placeholder text to indicate the type of input the user should fill.
 - [ ] The button should also have the type 'submit' to make the form work properly. Kindly add a type to the form button.